packages/augur-tools/src/flash/scripts.ts

```diff
@@ -760,6 +761,67 @@ export function addScripts(flash: FlashSession) {
     },
   });
 
+  flash.addScript({
+    name: 'get-market-order-book',
+    options: [
+      {
+        name: 'marketId',
+        abbr: 'm',
+        description: 'Show orders that have been placed on the book of this marketId'
+      }
+    ],
+    async call(this: FlashSession, args: FlashArguments) {
+      const result = await this.user.augur.getMarketOrderBook({ marketId: args.marketId as string});
+      this.log(JSON.stringify(result));
+      return result;
+    }
+  });
+
+  flash.addScript({
+    name: 'simple-orderbook-shaper',
+    options: [
+      {
+        name: 'marketIds',
+        abbr: 'm',
+        description: 'Market ids separated by commas for multiple to create orders and maintain order book, ie 0x122,0x333,0x4444'
+      },
+      {
+        name: 'userAccount',
+        abbr: 'u',
+        description: 'User account to create orders, if not provider contract owner is used'
+      },
+      {
+        name: 'meshEndpoint',
+        abbr: 'e',
+        required: false,
+        description: 'Mesh endpoint to connect'
+      }
+    ],
+    async call(this: FlashSession, args: FlashArguments) {
+      const marketIds = String(args.marketIds).split(',').map(id => id.trim());
+      const address = args.userAccount ? args.userAccount as string : null;
+      const endpoint = args.meshEndpoint ? String(args.meshEndpoint) : 'ws://localhost:60557';
+      const user: ContractAPI = await this.ensureUser(null, true, true, address, endpoint, true);
+      this.log('creating orderbook shaper instances');
+      const orderBooks = marketIds.map(m => new OrderBookShaper(m));
+
+      while(true) {
+        const timestamp = await this.user.getTimestamp();
+        for(let i = 0; i < orderBooks.length; i++) {
+          const orderBook: OrderBookShaper = orderBooks[i];
+          const marketId = orderBook.marketId;
+          const marketBook: MarketOrderBook = await this.user.augur.getMarketOrderBook({ marketId });
+          const orders = await orderBook.nextRun(marketBook, timestamp);
+          this.log(`creating ${orders.length} orders for ${marketId}`);
+          if (orders.length > 0) {
+            orders.map(order => user.placeZeroXOrder(order));
```

Comment:
`map` -> `forEach`

Reply:
don't want to deal with promises. it's a fire and forget.